container/single-instance/Dockerfile

```diff
@@ -17,11 +17,11 @@ LABEL org.opencontainers.image.created="%BUILDTIME%"
 # hadolint ignore=DL3037
 RUN zypper in -y openQA-single-instance openQA-bootstrap \
     qemu-arm qemu-ppc qemu-x86 qemu-tools sudo iputils os-autoinst-distri-opensuse-deps && \
-    zypper clean -a && \
-    sed -i -e 's/# service_port_delta =.*$/service_port_delta = 0/' /etc/openqa/openqa.ini
+    zypper clean -a
 EXPOSE 80 443 9526
 ENV skip_suse_specifics=1
 ENV skip_suse_tests=1
+ENV SERVICE_PORT_DELTA=0
```

Comment:
Why is this not an OPENQA_ variable?

Reply:
Good question. I think it should be one.

Reply:
Absolutely. Thank you! I only litterally replaced the variable name with an uppercase variant, haven't considered the prefix. Done